src/AdminDashboard/Settings/Security.js

```diff
@@ -180,65 +184,79 @@ class Security extends Component {
                             <div className="message">
-                                Delete browser data
-                                <div className="detail">Delete Web Storage and Indexed Database</div>
+                                {I18n.t('settings.delete_browser_data')}
+                                <div className="detail">
+                                    {I18n.t('settings.delete_web_storage_and_indexed_database')}
+                                </div>
                             </div>
                             <div className="controller">
-                                <button className="win-button" onClick={this.cleanWebStorage}>Delete</button>
+                                <button className="win-button" onClick={this.cleanWebStorage}>
+                                    { I18n.t('commons.delete') }
+                                </button>
                             </div>
                         </div>
 
                         <Confirmation 
-                            title="Delete browser data"
-                            message="Are you sure you want to delete all data/information in the web storage and indexed database?"
+                            title={I18n.t('settings.delete_browser_data')}
+                            message={I18n.t('settings.are_you_sure_you_want_to_delete_all_data/information_in_the_web_storage_and_indexed_database?')}
                             reference={el => this.deleteBrowserData = el} 
                         />
         
                         <div className="listElement">
                             <div className="message">
-                                Delete account
-                                <div className="detail">This action remove all information related to the user and the entity</div>
+                                { I18n.t('settings.delete_account') }
+                                <div className="detail">
+                                    { I18n.t('settings.this_action_remove_all_information_related_to_the_user_and_the_entity') }
+                                </div>
                             </div>
                             <div className="controller">
-                                <button className="win-button" onClick={this.deleteUser}>Delete</button>
+                                <button className="win-button" onClick={this.deleteUser}>
+                                    { I18n.t('commons.delete') }
+                                </button>
                             </div>
                         </div>
                         
                         <Confirmation 
-                            title="Account deletion"
-                            message="Are you certain to delete the account?"
+                            title={I18n.t('settings.account_deletion')}
+                            message={I18n.t('settings.are_you_certain_to_delete_the_account?')}
```

Comment:
It think you need to avoid special chars `?` 😶

Reply:
I think it is necessary for the correct translation in transifex :face_with_head_bandage:

Reply:
![image](https://user-images.githubusercontent.com/12489333/36385941-8b37ef8e-156a-11e8-8f0a-f2e4b867225d.png)

Reply:
If I delete the question mark in Transifex it will not be present either and it will not be a question but a statement